### ISSUE TITLE
Bug: ...Latest version (commit 5bf4717) stopped working

### ISSUE THREAD

**Issue by Frepke** [qdm12/private-internet-access-docker#187]
**TLDR**: *Container hangs on startup*

1. Is this urgent?

    - [x] Yes
    - [ ] No

2. What VPN service provider are you using?

    - [ ] PIA
    - [ ] Mullvad
    - [ ] Windscribe
    - [x] Surfshark
    - [ ] Cyberghost

3. What's the version of the program?

    **See the line at the top of your logs**

    `Running version latest built on 2020-07-08T23:43:31Z (commit 5bf4717)`

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

Below is the complete log, not only a part of it.

Logs:

```2020-07-09T06:34:02.252+0200	INFO	ShadowSocks version: 3.3.4
2020-07-09T06:34:02.291+0200	INFO	OpenVPN version: 2.4.9
2020-07-09T06:34:02.296+0200	INFO	Unbound version: 1.10.1
2020-07-09T06:34:02.300+0200	INFO	IPtables version: v1.8.4
2020-07-09T06:34:02.356+0200	INFO	TinyProxy version: 1.10.0
2020-07-09T06:34:02.356+0200	INFO	Settings summary below:
OpenVPN settings:
|--User: [redacted]
|--Password: [redacted]
|--Verbosity level: 0
|--Run as root: no
|--Surfshark settings:
 |--Network protocol: udp
 |--Region: netherlands amsterdam
 |--Target IP address: ��Y.�h
System settings:
|--User ID: 1000
|--Group ID: 100
|--Timezone: europe/amsterdam
|--IP Status filepath: /ip
DNS over TLS settings:
 |--DNS over TLS provider:
  |--quad9
  |--cloudflare
 |--Caching: enabled
 |--Block malicious: enabled
 |--Block surveillance: enabled
 |--Block ads: enabled
 |--Allowed hostnames:
  |--
 |--Private addresses:
  |--
 |--Verbosity level: 1/5
 |--Verbosity details level: 0/4
 |--Validation log level: 0/2
 |--IPv6 resolution: disabled
 |--Update: every 24h0m0s
Firewall settings:
 |--Allowed subnets: [redacted]/24
TinyProxy settings:
Port: 8888
 |--Authentication: enabled
 |--Log level: info
ShadowSocks settings:
 |--Password: [redacted]
 |--Log: enabled
 |--Port: 8388
 |--Method: chacha20-ietf-poly1305

2020-07-09T06:34:02.358+0200	INFO	openvpn configurator: checking for device /dev/net/tun
2020-07-09T06:34:02.359+0200	WARN	TUN device is not available: open /dev/net/tun: no such file or directory
2020-07-09T06:34:02.359+0200	INFO	openvpn configurator: creating /dev/net/tun
2020-07-09T06:34:02.359+0200	INFO	routing: detecting default network route
2020-07-09T06:34:02.360+0200	INFO	routing: default route found: interface eth0, gateway 172.17.0.1, subnet 172.17.0.0/16
2020-07-09T06:34:02.360+0200	INFO	firewall configurator: accepting all traffic
2020-07-09T06:34:02.370+0200	INFO	routing: adding 10.54.1.0/24 as route via eth0
2020-07-09T06:34:02.370+0200	INFO	Launching standard output merger
2020-07-09T06:34:02.379+0200	INFO	firewall configurator: clearing all rules
2020-07-09T06:34:02.384+0200	INFO	firewall configurator: blocking all traffic
2020-07-09T06:34:02.400+0200	INFO	firewall configurator: creating general rules
2020-07-09T06:34:02.442+0200	INFO	firewall configurator: allowing output traffic to VPN server 89.46.223.104 through eth0 on port udp 1194
2020-07-09T06:34:02.463+0200	INFO	firewall configurator: accepting input and output traffic for 172.17.0.0/16
2020-07-09T06:34:02.472+0200	INFO	firewall configurator: accepting input traffic through eth0 from 10.54.1.0/24 to 172.17.0.0/16
2020-07-09T06:34:02.480+0200	INFO	firewall configurator: accepting output traffic through eth0 from 172.17.0.0/16 to 10.54.1.0/24
2020-07-09T06:34:02.486+0200	INFO	dns configurator: using DNS address 9.9.9.9 internally
2020-07-09T06:34:02.486+0200	INFO	dns configurator: using DNS address 9.9.9.9 system wide

```

Configuration file:

```yml

```

Host OS: Debian 10

**Comment by Frepke** [comment]
After a while (hours?) there's some action in the logfile:

```2020-07-09T11:21:40.368+0200	ERROR	ip getter: Get "https://ifconfig.me/ip": dial tcp: lookup ifconfig.me on 162.252.172.57:53: write udp 172.17.0.2:39832->162.252.172.57:53: write: operation not permitted


2020-07-09T11:21:40.368+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:21:45.370+0200	ERROR	ip getter: Get "https://ip4.ddnss.de/meineip.php": dial tcp: lookup ip4.ddnss.de on 162.252.172.57:53: write udp 172.17.0.2:45011->162.252.172.57:53: write: operation not permitted


2020-07-09T11:21:45.370+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:21:50.372+0200	ERROR	ip getter: Get "https://diagnostic.opendns.com/myip": dial tcp: lookup diagnostic.opendns.com on 162.252.172.57:53: write udp 172.17.0.2:41037->162.252.172.57:53: write: operation not permitted


2020-07-09T11:21:50.372+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:21:55.374+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:36649->162.252.172.57:53: write: operation not permitted


2020-07-09T11:21:55.374+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:00.376+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:55298->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:00.376+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:05.378+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:60489->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:05.378+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:10.380+0200	ERROR	ip getter: Get "https://ipinfo.io/ip": dial tcp: lookup ipinfo.io on 162.252.172.57:53: write udp 172.17.0.2:33731->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:10.380+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:15.382+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:48771->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:15.382+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:20.384+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:33762->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:20.384+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:25.386+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:54614->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:25.386+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:30.388+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:44036->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:30.389+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:35.391+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:42275->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:35.391+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:40.393+0200	ERROR	ip getter: Get "https://ifconfig.me/ip": dial tcp: lookup ifconfig.me on 162.252.172.57:53: write udp 172.17.0.2:45631->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:40.393+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:45.395+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:35377->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:45.395+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:50.397+0200	ERROR	ip getter: Get "https://ipinfo.io/ip": dial tcp: lookup ipinfo.io on 162.252.172.57:53: write udp 172.17.0.2:44082->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:50.397+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:22:55.399+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:36574->162.252.172.57:53: write: operation not permitted


2020-07-09T11:22:55.399+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:00.400+0200	ERROR	ip getter: Get "https://api.ipify.org": dial tcp: lookup api.ipify.org on 162.252.172.57:53: write udp 172.17.0.2:52022->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:00.401+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:05.403+0200	ERROR	ip getter: Get "https://api.ipify.org": dial tcp: lookup api.ipify.org on 162.252.172.57:53: write udp 172.17.0.2:39078->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:05.403+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:10.405+0200	ERROR	ip getter: Get "https://ifconfig.me/ip": dial tcp: lookup ifconfig.me on 162.252.172.57:53: write udp 172.17.0.2:57887->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:10.405+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:15.407+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:54052->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:15.407+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:20.409+0200	ERROR	ip getter: Get "https://ipinfo.io/ip": dial tcp: lookup ipinfo.io on 162.252.172.57:53: write udp 172.17.0.2:55221->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:20.409+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:25.411+0200	ERROR	ip getter: Get "https://domains.google.com/checkip": dial tcp: lookup domains.google.com on 162.252.172.57:53: write udp 172.17.0.2:46687->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:25.411+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:30.415+0200	ERROR	ip getter: Get "https://ifconfig.me/ip": dial tcp: lookup ifconfig.me on 162.252.172.57:53: write udp 172.17.0.2:50753->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:30.415+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:35.417+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:53463->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:35.418+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:40.420+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:43869->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:40.420+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:45.422+0200	ERROR	ip getter: Get "https://domains.google.com/checkip": dial tcp: lookup domains.google.com on 162.252.172.57:53: write udp 172.17.0.2:53942->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:45.422+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:50.423+0200	ERROR	ip getter: Get "https://domains.google.com/checkip": dial tcp: lookup domains.google.com on 162.252.172.57:53: write udp 172.17.0.2:59054->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:50.424+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:23:55.426+0200	ERROR	ip getter: Get "https://ip4.ddnss.de/meineip.php": dial tcp: lookup ip4.ddnss.de on 162.252.172.57:53: write udp 172.17.0.2:35496->162.252.172.57:53: write: operation not permitted


2020-07-09T11:23:55.426+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:00.427+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:49568->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:00.427+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:05.429+0200	ERROR	ip getter: Get "https://domains.google.com/checkip": dial tcp: lookup domains.google.com on 162.252.172.57:53: write udp 172.17.0.2:42618->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:05.430+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:10.431+0200	ERROR	ip getter: Get "https://api.ipify.org": dial tcp: lookup api.ipify.org on 162.252.172.57:53: write udp 172.17.0.2:40271->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:10.432+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:15.433+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:56753->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:15.434+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:20.435+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:33623->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:20.436+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:25.438+0200	ERROR	ip getter: Get "https://api.ipify.org": dial tcp: lookup api.ipify.org on 162.252.172.57:53: write udp 172.17.0.2:45951->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:25.438+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:30.440+0200	ERROR	ip getter: Get "https://diagnostic.opendns.com/myip": dial tcp: lookup diagnostic.opendns.com on 162.252.172.57:53: write udp 172.17.0.2:48257->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:30.440+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:35.442+0200	ERROR	ip getter: Get "https://diagnostic.opendns.com/myip": dial tcp: lookup diagnostic.opendns.com on 162.252.172.57:53: write udp 172.17.0.2:49575->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:35.442+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:40.444+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:38938->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:40.444+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:45.446+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:37608->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:45.446+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:50.448+0200	ERROR	ip getter: Get "https://diagnostic.opendns.com/myip": dial tcp: lookup diagnostic.opendns.com on 162.252.172.57:53: write udp 172.17.0.2:57191->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:50.448+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:24:55.450+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:59628->162.252.172.57:53: write: operation not permitted


2020-07-09T11:24:55.450+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:00.452+0200	ERROR	ip getter: Get "https://ifconfig.me/ip": dial tcp: lookup ifconfig.me on 162.252.172.57:53: write udp 172.17.0.2:48790->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:00.452+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:05.454+0200	ERROR	ip getter: Get "https://ifconfig.me/ip": dial tcp: lookup ifconfig.me on 162.252.172.57:53: write udp 172.17.0.2:40140->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:05.454+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:10.456+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:48356->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:10.456+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:15.458+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:50327->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:15.458+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:20.460+0200	ERROR	ip getter: Get "https://diagnostic.opendns.com/myip": dial tcp: lookup diagnostic.opendns.com on 162.252.172.57:53: write udp 172.17.0.2:39434->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:20.460+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:25.462+0200	ERROR	ip getter: Get "https://ifconfig.me/ip": dial tcp: lookup ifconfig.me on 162.252.172.57:53: write udp 172.17.0.2:54587->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:25.462+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:30.467+0200	ERROR	ip getter: Get "http://ip1.dynupdate.no-ip.com:8245": dial tcp: lookup ip1.dynupdate.no-ip.com on 162.252.172.57:53: write udp 172.17.0.2:37010->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:30.468+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:35.470+0200	ERROR	ip getter: Get "https://api.ipify.org": dial tcp: lookup api.ipify.org on 162.252.172.57:53: write udp 172.17.0.2:53942->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:35.470+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:40.472+0200	ERROR	ip getter: Get "https://ifconfig.me/ip": dial tcp: lookup ifconfig.me on 162.252.172.57:53: write udp 172.17.0.2:56439->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:40.472+0200	INFO	ip getter: retrying in 5 seconds


2020-07-09T11:25:45.474+0200	ERROR	ip getter: Get "https://ifconfig.io/ip": dial tcp: lookup ifconfig.io on 162.252.172.57:53: write udp 172.17.0.2:49151->162.252.172.57:53: write: operation not permitted


2020-07-09T11:25:45.474+0200	INFO	ip getter: retrying in 5 seconds

**Comment by Frepke** [comment]
For now, back to an older commet :)

**Comment by WoLpH** [comment]
Yes, getting the same. The auth file isn't written anymore

**Comment by qdm12** [comment]
Yes sorry about that openvpn doesn't even start. It's strange on my local machine it works but not on my linux server 😕  I'll fix it in the coming minutes nonetheless. Thanks for reporting it!

**Comment by qdm12** [comment]
Done 😉  It was me waiting for an event instead of triggering the event, for both shadowsocks and tinyproxy. So the container was deadlocking 😕  Anyway it's fixed thanks guys for letting me know 🎉
